zerver/tests/test_alert_words.py

```diff
@@ -145,23 +149,50 @@ def test_json_list_add(self) -> None:
         )
         self.assert_json_error(result, "alert_words[0] is too long (limit: 100 characters)")
 
+
     def test_json_list_remove(self) -> None:
         user = self.get_user()
         self.login_user(user)
 
-        result = self.client_post(
+        self.client_post(
             "/json/users/me/alert_words",
             {"alert_words": orjson.dumps(["one", "two", "three"]).decode()},
         )
-        response_dict = self.assert_json_success(result)
-        self.assertEqual(set(response_dict["alert_words"]), {"one", "two", "three"})
 
         result = self.client_delete(
-            "/json/users/me/alert_words", {"alert_words": orjson.dumps(["one"]).decode()}
+            "/json/users/me/alert_words",
+            {"alert_words": orjson.dumps(["one"]).decode()},
```

Comment:
minor: was the addition of the comma here intentional?  If so, it'd be good to add a comment to the review (or in the code - your choice) explaining why this was needed.